lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -472,8 +472,11 @@ sub stat_sensitivity_exact {
     my $results = $self->get_rawstats();
 
     my $total_true = $self->stat_total_true_positive_exact;
-    if ($total_true == 0) {
-        return 0;
+    {
+        no warnings;
```

Comment:
The variable may not necessarily be a numerical zero.  It could be `undef`; a string like `NaN`, `nan`, or `NA`, or some other potentially non-numerical result. Unfortunately, I don't exactly know what the other non-numerical results could be at the moment, but I know that perl will evaluate it to be zero which is what I would like the code to do.